android/gradle.properties

```diff
@@ -1,5 +1,5 @@
 android.useAndroidX=true
-Purchases_kotlinVersion=1.4.0
+Purchases_kotlinVersion=1.5.31
```

Comment:
This is required now by [RN 0.67](https://reactnative.dev/blog/2022/01/19/version-067).